src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -76,6 +77,7 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 _gen1SizeCounter ??= new PollingCounter("gen-1-size", this, () => GC.GetGenerationSize(1)) { DisplayName = "Gen 1 Size", DisplayUnits = "B" };
                 _gen2SizeCounter ??= new PollingCounter("gen-2-size", this, () => GC.GetGenerationSize(2)) { DisplayName = "Gen 2 Size", DisplayUnits = "B" };
                 _lohSizeCounter ??= new PollingCounter("loh-size", this, () => GC.GetGenerationSize(3)) { DisplayName = "LOH Size", DisplayUnits = "B" };
+                _pohSizeCounter ??= new PollingCounter("poh-size", this, () => GC.GetGenerationSize(4)) { DisplayName = "POH Size", DisplayUnits = "B" };
```

Comment:
Is POH a well-enough understood acronym?  Will we document that as pinned object heap anywhere?

Reply:
https://devblogs.microsoft.com/dotnet/announcing-net-5-0-preview-2/ announced it as POH (Pinned Object Heap). I could add the "(Pinned Object Heap)" part and keep it around until it is a more well-understood acronym by our customers.

Reply:
Note we are also using `UOH` in number of places that means `LOH` or `POH`.

I had a similar reaction to these TLAs. Some of the discussion is captured in https://github.com/dotnet/runtime/issues/33445